src/cli/app.js

```diff
@@ -109,6 +110,17 @@ async function mcodeApp(Client, fromDate, toDate, pathToConfig, pathToRunLogs, d
       }
     }
 
+    // check if config specifies that MRN needs to be masked
+    // if it does need to be masked, mask all references to MRN outside of the patient resource
+    const patientConfig = config.extractors.filter((e) => e.type === 'CSVPatientExtractor')[0];
```

Comment:
```suggestion
    const patientConfig = config.extractors.find((e) => e.type === 'CSVPatientExtractor');
```

`find` instead of `filter`[0] probably makes sense here. `find` will return the first occurrence that matches the boolean you provide, and `undefined` if not found